src/Sulu/Bundle/AdminBundle/Resources/js/components/Navigation/ProfileButton.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {action, observable} from 'mobx';
+import classNames from 'classnames';
+import Icon from '../Icon';
+import Button from '../Button';
+import {translate} from '../../utils';
+import profileButtonStyles from './profileButton.scss';
+
+type Props = {
+    onLogoutClick: () => void,
+    onProfileClick: () => void,
+    suluVersion: string,
+    suluVersionLink: string,
+    userImage: ?string,
+    username: string,
+}
+
+@observer
+class ProfileButton extends React.Component<Props> {
```

Comment:
not sure about the name - think this is more about the user. what do you think about `UserSection`/`UserPanel`/`UserAccordion`?